app/builders/digest_email_builder.rb

```diff
@@ -1,81 +1,76 @@
 class DigestEmailBuilder < ApplicationBuilder
-  def initialize(address:, digest_items:, digest_run:, subscriber_id:)
+  def initialize(address:, digest_item:, digest_run:, subscriber_id:)
     @address = address
-    @digest_items = digest_items
+    @digest_item = digest_item
     @digest_run = digest_run
     @subscriber_id = subscriber_id
   end
 
   def call
     Email.create!(
       address: address,
-      subject: I18n.t!("emails.digests.#{digest_run.range}.subject"),
+      subject: I18n.t!(
+        "emails.digests.#{digest_run.range}.subject",
+        title: digest_item.subscriber_list_title,
```

Comment:
Do we know if there are any character limits on this and whether any of our titles may blow it? I guess we run a similar risk with content changes but they don't use the list title and I know we have some crazy long list titles.

Reply:
Oh, interesting idea. Our maximum title is 2163 characters, which is fine. I tried with longer titles - eventually getting an OOM error at 1 billion trillion (or something). Should be fine!

Reply:
Is that with Notify? I wondered if SES or Notify imposed any limits. The StackOverflow question I can find on it seems there's just vague recommendations: https://stackoverflow.com/questions/1592291/what-is-the-email-subject-length-limit

Reply:
Oh sorry, I was being dim (I thought you meant with I18n). I've had a go and 2163 chars is fine. Seems to be fine up to 10K chars. I also chatted with David from GOV.UK Notify. They don't know what the actual limit is, as there is indeed nothing written about it. But for now, I don't think it'll cause any problems - even for our insanely long titles.